tests/imageCarousel.test.js

```diff
@@ -0,0 +1,16 @@
+import { configure, shallow } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import React from 'react';
+import { expect } from 'chai';
+
+import ImageCarousel from '../client/src/components/ImageCarousel.jsx';
+import DishPhotos from '../client/src/components/DishPhotos.jsx';
+
+configure({ adapter: new Adapter() });
```

Comment:
configuration can be done at a setupTests.js file at the root of your repo to avoid having to write in each test file. jest will automatically look for a file named setupTests and use the configuration from there

Reply:
I tried making a file to separate the configuration, but Jest is giving me an error that I need to import on the specific test file. I tried calling it different names and moved it globally too and doesn't work.